third_party/3/jwt.pyi

```diff
@@ -0,0 +1,36 @@
+from typing import Mapping, Any, Optional, Union
+
```

Comment:
From https://github.com/jpadilla/pyjwt/blob/master/jwt/__init__.py it looks like there's a few more public functions. The documentation isn't very comprehensive, but at least `register_algorithm` is documented.

It's important to include all public functions, because otherwise people using those functions will get spurious errors.